OpenOversight/app/templates/list_unit.html

```diff
@@ -0,0 +1,190 @@
+{% extends "base.html" %}
```

Comment:
borrowed almost entirely from https://github.com/lucyparsons/OpenOversight/blob/develop/OpenOversight/app/templates/officer.html

Reply:
maybe I change this filename to `unit.html` to match the url path ?